teuthology/task/buildpackages/common.sh

```diff
@@ -41,9 +41,6 @@ function get_ceph() {
 
     test -d ceph || git clone ${git_ceph_url} ceph
     cd ceph
-    if test -d src ; then # so we don't try to fetch when using a fixture
-       git fetch --tags http://github.com/ceph/ceph
-    fi
```

Comment:
Yes, I understand, but for example in SUSE/ceph we control the tags closely and in the past there was already a situation where we purposefully did not push an upstream tag in order to keep the version number fixed.